Source/EasyNetQ/Scheduling/DeadLetterExchangeAndMessageTtlScheduler.cs

```diff
@@ -52,7 +52,9 @@ public async Task FuturePublishAsync<T>(T message, TimeSpan delay, string topic
             {
                 Properties =
                 {
-                    DeliveryMode = messageDeliveryModeStrategy.GetDeliveryMode(typeof(T))
+                    DeliveryMode = messageDeliveryModeStrategy.GetDeliveryMode(typeof(T)),
+                    PriorityPresent = priority.HasValue,
+                    Priority = priority ?? 0
```

Comment:
probably this is not very good since
![image](https://user-images.githubusercontent.com/21261007/65546127-85022000-df1f-11e9-8d32-075eee4528e3.png)